index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav class="top-navigation">
+            <div class="welcome-message">Welcome</div>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
```

Comment:
- [ ] Here I noticed that your logo “Welcome” doesn’t have an (‘a’ tag ). This is needed for three reasons: convention, semantics, and user experience(UX). Check this [link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :wink:  So please do well to add it to your logo and in the` herf,` add a link to the homepage. :100: